app/javascript/lib/validation/validate-file-content.js

```diff
@@ -276,99 +279,138 @@ function validateClusterCoordinates(headers) {
   return issues
 }
 
-/** Get properties about this validation run to log to Mixpanel */
-function getLogProps(fileObj, fileType, errorObj) {
-  const { file, table, delimiter } = fileObj
-  const { errors, summary } = errorObj
+/** parse a metadata file, and return an array of issues, along with file parsing info */
+export async function parseMetadataFile(chunker, mimeType) {
+  const { parsedHeaders, delimiter } = await getParsedHeaderLines(chunker, mimeType, 2)
```

Comment:
Let's call this `headers`.  Assuming it's parsed is reasonable, like assuming it's a header for an SCP study file.  Using this name would also let us consistently refer to "headers" as parsed headers throughout this high-level module, rather than "parsedHeaders" in some functions and "headers" in other functions.

```suggestion
  const { headers, delimiter } = await getParsedHeaderLines(chunker, mimeType, 2)
```